waspc/data/Generator/templates/react-app/src/operations/resources.js

```diff
@@ -45,3 +45,22 @@ export const invalidateQueriesUsing = (resources) => {
     queryClient.invalidateQueries(queryCacheKey)
   }
 }
+
+export function removeQueries() {
+  // TODO(filip): We are currently removing all the queries, but we should
+  // remove only non-public, user-dependent queries - public queries are
+  // expected not to change in respect to the currently logged in user.
```

Comment:
This comments doesn't make sense in this context -> you should probably move it to the login function.

Reply:
We currently only do this in the login function, but moving the comment to the call site seems a bit off to me. What if we were using it in more than one place.

Ideally, I'd name the function `removePrivateQueryData` and leave the correct implementation with a TODO.

Reply:
Ok great - resolve this one then pls when you are done!